parl/remote/remote_decorator.py

```diff
@@ -0,0 +1,255 @@
+#   Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pyarrow
+import threading
+import time
+import zmq
+from parl.remote import remote_constants
+from parl.utils import get_ip_address, logger, to_str, to_byte
+from parl.utils.exceptions import SerializeError, DeserializeError
+from parl.utils.communication import loads_argument, dumps_return
+"""
+Three steps to create a remote class:
+1. add a decroator(@parl.remote) before the definition of the class;
+2. create an instance of remote class;
```

Comment:
create an instance of remote class with the information of server_ip and server_port

Reply:
server_ip and server_port won't be used in creating instance, they will be used in as_remote function.